data/map planets.txt

```diff
@@ -3710,6 +3717,7 @@ planet Sandsnap
 	attributes hai station uninhabited
 	landscape land/station40
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
+	description `	After the opening of the wormhole in Waypoint, Sandsnap was briefly considered as a possible destination for humans looking to settle in Hai space. Reopening the station would have provided ample room for millions of potential migrants, without disturbing planetary ecosystems. Ultimately it was decided that the slow trickle of human settlers to Hai space did not justify the costs of bringing Sandsnap back online, and that it would be better for both humans and Hai if the two species weren't sequestered off from each other.`
```

Comment:
```suggestion
	description `	A tourist FAQ in the docking area cheerily informs any visitors that after the opening of the wormhole in Waypoint, Sandsnap was briefly considered as a possible destination for humans looking to settle in Hai space. Reopening the station would have provided ample room for millions of potential migrants, without disturbing planetary ecosystems. Ultimately it was decided that the slow trickle of human settlers to Hai space did not justify the costs of bringing Sandsnap back online, and that it would be better for both humans and Hai if the two species weren't sequestered off from each other.`
```

Reply:
I really like this angle with a quick nod to how we learned the info.

It's the type of thing that doesn't need constant repeating across every instance to still set  up plausible expectation of similar methods. :)